### PR TITLE
Husky, Fix Deposit tests, linter and prettier

### PR DESCRIPTION
**Pull request by debrayperalta**
The SDK is growing, and in order to ensure quality in the code some pre-commit hooks were added to decrease the probability that we may commit broken code (a test did not pass, something was not formatted, the linter threw a warning).

In this PR the next changes were made

- Husky was installed
- We now have a pre commit hook
- The pre commit hook will format the code, lint it and finally run the tests
- The deposit tests were broken, so I fixed them
- There was an useless deposit test, I deleted it
- The project was linted and formatted to pass the husky check

**EDIT**: Some changes were made, see [this comment](https://github.com/rsksmart/lumino-light-client-sdk/pull/84#issuecomment-688499795)

### CODE REVIEW

**Comment by mortelli** [comment]
Concept ACK, superb idea :+1:

File: src/Lumino/index.js [hunk 1]
```diff
@@ -82,8 +83,9 @@ const Lumino = () => {
   const getRNSOptions = () => {
     return {
       networkId: this.chainId,
+      // TODO: Implement this
       contractAddresses: {
-        registry: rifConfig.rns.contracts.rns,
+        registry: AddressZero,
```
Comment (marcosmartinez7) [inline src/Lumino/index.js:88]: Dont understand this, why removing the rifConfig param?

Comment (debrayperalta) [reply src/Lumino/index.js:88]: This was undefined, rifConfig was a .ENV variable, so the SDK doesn't hold any reference to it.
I will fix this with the configParams.registryAddress

Comment (debrayperalta) [reply src/Lumino/index.js]: Fixed with a new commit, the address is now in luminoConfig

**Comment by debrayperalta** [comment]
I made some changes in the husky hooks

- The pre-commit hook will only format with prettier and lint the code
- The pre-push hook will do all of the above plus running the tests

Why? The tests can take up to 10s or more sometimes, and making small commits may be too much of a hassle because of this, prettier and eslint run much faster.

I will edit the description as well

**Review by marcosmartinez7 — APPROVED** [review]

**Pull request merged**